### PR TITLE
Refactorings of code done while working on POC for Inventor Configurator CPQ integration

### PR DESCRIPTION
**Pull request by msimko81**

### CODE REVIEW

File: WebApplication/Controllers/ProjectsController.cs [hunk 1]
```diff
@@ -79,7 +81,7 @@ public async Task<IEnumerable<ProjectDTO>> ListAsync()
         }
 
         [HttpPost]
-        public async Task<ActionResult<ProjectDTO>> CreateProject([FromForm]NewProjectModel projectModel)
+        public async Task<ActionResult<string>> CreateProject([FromForm]NewProjectModel projectModel)
```
Comment (erymski) [inline WebApplication/Controllers/ProjectsController.cs:84]: Should not it cause changes on client-side? (or maybe it's not used at all?)

File: WebApplication/Services/ProjectService.cs [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+            _logger.LogInformation($"'TransferData' for {projectUrl} is done.");
+
+            return signedUrl;
+        }
+        
+        /// <summary>
+        /// Get list of project names for a bucket.
+        /// </summary>
+        public async Task<ICollection<string>> GetProjectNamesAsync(OssBucket bucket = null)
```
Comment (erymski) [inline WebApplication/Services/ProjectService.cs:65]: @hradilm Looks like the refactoring for project enumeration is already done 🎉

File: WebApplication/Controllers/ProjectsController.cs (hunk 1 above)
Comment (mivasi) [reply WebApplication/Controllers/ProjectsController.cs:84]: ProjectDTO was mistake, it was always returning string. Just mistakenly mentioned in function definition that it returns ProjectDTO

Comment (msimko81) [reply WebApplication/Controllers/ProjectsController.cs:84]: Actually I have just fixed the method signature. The type of returned payload was string prior my changes.

**Review by erymski — APPROVED** [review]

**Pull request merged**